src/cli/commands/dev.ts

```diff
@@ -71,12 +71,18 @@ export class Dev implements Command {
     }
 
     const startModule = createStartModuleContent({
+      registerTypeScript: {
+        ...layout.tsConfig.content.options,
+        target: ts.ScriptTarget.ES5,
```

Comment:
What's the reasoning for forcing target ES5? Eventually, in build, I assume we will follow the real tsconfig options. And I'm afraid it might create surprises where things works in dev but not on build or vice-versa

Reply:
Agree

Actually, in dev, I do not see the point of these overrides like we have for data mode app runs (where node needs to run app in sub process).